src/plotting/plotSettings/moreOptions/moreOptionsToggle.component.tsx

```diff
@@ -35,7 +35,7 @@ const MoreOptionsToggle = (props: MoreOptionsProps) => {
             top: -95,
```

Comment:
Can we change `top: -95` to `bottom: -18`? This makes it so the popup menu appears mostly above the channel we're editing, which stops any overflow issues